app/albums/create/page.tsx

```
@@ -1,123 +1,47 @@
-
-		reader.readAsDataURL(e.target.files[0]);
-	};
-
 	const onSubmit: SubmitHandler<Inputs> = async (data) => {
-		const createdAt = dayjs().format("YYYY-MM-DD");
-
-		if (!data.title || inputCoverImg === null) {
-			return;
-		}
+		try {
+			const documentData: any = {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)